<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWindow</class>
 <widget class="QDialog" name="BrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetBottom" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
        <item row="1" column="0" colspan="4">
         <widget class="QPushButton" name="pushShow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Show</string>
          </property>
          <property name="shortcut">
           <string>Space</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QCheckBox" name="checkHideTreeview">
          <property name="text">
           <string>Hide word list</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">4</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">5</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">6</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="pushPrevious">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
          <property name="shortcut">
           <string>Left</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="checkShowDefinitionDirectly">
          <property name="text">
           <string>Show definition directly</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QPushButton" name="pushNext">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
          <property name="shortcut">
           <string>Right</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
